### ISSUE TITLE
Damage Text - Undeclared Globals

### ISSUE THREAD

**Issue by Devalut** [BribeFromTheHive/DamageEngine#5]
Note both depicted instances are from the same map where a peasant attacks a herobloodmage (note: Without the demo trigger that would modify the peasant's damage.)

![image](https://user-images.githubusercontent.com/20247202/215349216-2b638077-3306-46b9-b9c8-fd8b5a4e324e.png)
Above is what I posted originally [on the debug utilities page](https://www.hiveworkshop.com/threads/debug-utils-ingame-console-etc.330758/post-3567778), but I didn't install the LUA version of Damage engine from here (Github) so perhaps the above issue is using the one from the hive.

![image](https://user-images.githubusercontent.com/20247202/215349757-499671ab-c785-4e5c-adb6-b176772c915d.png)
Minor success, above is  from using the damage engine provided from here (Github)

Thank you for taking time to help me with this.

**Comment by BribeFromTheHive** [comment]
I am going to be consolidating my GitHub soon so as to be less confusing, but for now I recommend getting my resources from https://github.com/BribeFromTheHive/Lua/tree/master . Could you please check if that's giving you the same behavior? I don't think I am referencing that GUI variable anymore.

**Comment by Devalut** [comment]
I can't seem to launch it,  do you mind giving a step by step for how I can install it?
I've crashed my editor four times now and I'm losing my fruitypebbles.

**Comment by BribeFromTheHive** [comment]
Yeah if it didn't compile then I am 99% sure I did something wrong. That stuff is so bleeding edge that it needs to first go to a hospital. I'll look into it and the others in the next weeks.

**Comment by Devalut** [comment]
Happy to be even a sliver of help!
